.eslintrc.cjs

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  extends: [`mixpanel`],
+  env: {
+    node: true,
+    browser: true,
+  },
+  rules: {
+    'space-before-function-paren': `off`,
+    // in some dom operation, while(true) is necessary
```

Comment:
we shouldn't disable this rule globally, just in the one spot where we need it

Reply:
oh i see 👍